src/utils/DawnHelpers.cpp

```diff
@@ -25,46 +25,53 @@
 
 namespace utils {
 
+    namespace {
+
+        shaderc_shader_kind ShaderCShaderKind(dawn::ShaderStage stage) {
```

Comment:
nit: seems to be capitalized as Shaderc rather than ShaderC

Reply:
Done